extensions/adobe/experience/decisioning/proposition-detail.schema.json

```diff
@@ -0,0 +1,68 @@
+{
+  "meta:license": [
+    "Copyright 2019 Adobe Systems Incorporated. All rights reserved.",
+    "This work is licensed under a Creative Commons Attribution 4.0 International (CC BY 4.0) license",
+    "you may not use this file except in compliance with the License. You may obtain a copy",
+    "of the License at https://creativecommons.org/licenses/by/4.0/"
+  ],
+  "$id": "https://ns.adobe.com/experience/decisioning/proposition-detail",
+  "$schema": "http://json-schema.org/draft-06/schema#",
+  "title": "Decision Proposition",
+  "type": "object",
+  "meta:abstract": true,
+  "meta:extensible": true,
+  "description": "The  Decision Proposition XDM is used to capture the outcome of a decisioning process",
+  "definitions": {
+    "proposition-detail-datatype": {
+      "properties": {
+        "xdm:activity": {
+          "title": "Decision Activity",
+          "description": "Snapshot of the activity for which the proposition was made. The object contains at least the @id property value which is the URI of the decision activity. See schema https://ns.adobe.com/experience/decisioning/activity",
+          "$ref": "https://ns.adobe.com/xdm/data/record"
```

Comment:
Can you explain the intent of using data/record here? This schema was not designed to be used like this. What do you really need it for? @kstreeter  can you look at this as well please?

Reply:
We have not yet introduced the class of Activity with this PR. What the reference to data/record does here is to create a stub. We want to say at a minimum that this "activity" is more than a JSON Object because it is an Entity with an '@id' and has optionally time series events associated. This is closer to what an "activity" is supposed to be.

Reply:
I decided to introduce the minimal stub classes for Activities and Options